### PR TITLE
Add draft version of multi-line grammar

### PR DESCRIPTION
**Pull request by julia-sprenger**
Tests are still failing.

### CODE REVIEW

**Comment by julia-sprenger** [comment]
@gschnabel Voilà, now you can build comments on top of those changes ;)

File: src/larktools/ebnf_grammar.py
Comment (gschnabel) [inline src/larktools/ebnf_grammar.py:1]: I think we should keep `arith_expr` on the right-hand side of an assignment.

File: src/larktools/ebnf_grammar.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   start: assign_var
 
-  assign_var: VARNAME "=" arith_expr
```
Comment (gschnabel) [inline src/larktools/ebnf_grammar.py:18]: I think we should keep `arith_expr` on the right-hand side of an assignment.

Comment (julia-sprenger) [reply src/larktools/ebnf_grammar.py:18]: right, expressions across multiple lines wouldn't also work yet

File: src/larktools/ebnf_grammar.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
   start: assign_var
 
-  assign_var: VARNAME "=" arith_expr 
+  assign_var: VARNAME "=" multi_line_block
```
Comment (julia-sprenger) [inline src/larktools/ebnf_grammar.py]: revert to
```suggestion
  assign_var: VARNAME "=" arith_expr
```

**Comment by julia-sprenger** [comment]
Hi @gschnabel I think I fixed all remaining issues on this. If you are happy with it feel free to merge.

**Pull request merged**